selfdrive/test/docker_build.sh

```diff
@@ -28,9 +28,10 @@ fi
 export DOCKER_REGISTRY=ghcr.io/commaai
 export COMMIT_SHA=$(git rev-parse HEAD)
 
+ARCH=$(uname -m)
 LOCAL_TAG=$DOCKER_IMAGE
-REMOTE_TAG=$DOCKER_REGISTRY/$LOCAL_TAG
-REMOTE_SHA_TAG=$REMOTE_TAG:$COMMIT_SHA
+REMOTE_TAG=$DOCKER_REGISTRY/$LOCAL_TAG:$ARCH
```

Comment:
should arch be a part of the tag? my thought was it would be part of the name, but not sure about that.
maybe openpilot-base becomes the multiarch one, and openpilot-base-arm64 and openpilot-base-amd64 are separate images

Reply:
thats a good point, it's less confusing that way